src/dojos/cd-find-dojo.vue

```diff
@@ -169,14 +187,32 @@
       color: @cd-white;
       margin: 0 -16px;
 
+      &--reduced {
+        display: flex;
+        background: @cd-green;
+        color: @cd-white;
+        margin: 0 -16px;
+        max-height: 200px;
```

Comment:
As per my other comment on these classes, this should just be `max-height: 200px;` with nothing else